### PR TITLE
Feat: Alert

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
Three types of alerts are added 

- Simple alerts
- Alerts with Icons
- Alert with dismiss functionality

Preview link: https://deploy-preview-8--hydrogen-ui.netlify.app/docs/alert.html

### CODE REVIEW

**Review by adith-a-danthi — COMMENTED** [review]
LGTM! Great work @junaidshaikh-js!

File: components/alert.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  background-color: var(--primary-alert);
+  border-color: #2a7fff;
+  color: #0e2e5e;
+}
+
+.alert-secondary {
+  background-color: var(--secondary-alert);
+  border-color: #919191;
+  color: #3d3131;
+}
```
Comment (Maianki) [inline components/alert.css:19]: Use CSS variables for color.

**Review by Maianki — CHANGES_REQUESTED** [review]
Suggested some changes. Everything else looks good.

File: components/alert.css (hunk 1 above)
Comment (junaidshaikh-js) [reply components/alert.css:19]: They are just lighter and darker versions of the background color, hence variables aren't used.

**Pull request merged**